src/index.js

```diff
@@ -30,7 +29,7 @@ class RpcEngine {
 
   _handle (_req, cb) {
     // clone request object
-    const req = clone(_req)
```

Comment:
`clone` is a deep clone, spread is not. Is this a concern?

Reply:
I think we should be OK: maintaining nested object references _after_ sending a transaction seems like something we don't need to support. Nested object values should just be able to be copied.

Reply:
sounds like bugs and zalgo to me

Reply:
I wonder how JSON stringify reacts to their broken arrays